marklogic-data-hub-central/ui/src/components/results-tabular-view/results-tabular-view.tsx

```diff
@@ -360,19 +361,44 @@ const ResultsTabularView = (props) => {
       if (entity.name === props.selectedEntities[0]) {
         entity.primaryKey && setPrimaryKey(entity.primaryKey);
       }
+      setExpandedNestedTableRows([]);
     }));
   }, [props.selectedEntities, searchOptions.selectedTableProperties]);
 
   const expandedRowRender = (rowId) => {
     const nestedColumns = [
-      {title: "Property", dataIndex: "property", width: "33%"},
-      {title: "Value", dataIndex: "value", width: "34%"},
-      {title: "View", dataIndex: "view", width: "33%"},
+      {
+        text: "Property",
+        title: "Property",
+        dataIndex: "property",
```

Comment:
These properties should be changed from `dataIndex` > `dataField`

Reply:
@mrodrigu-marklogic can you check again